### DISCUSSION TITLE
How to make multilingual site ?

### DISCUSSION THREAD

**Discussion by ensyvz** [mmistakes/so-simple-theme#389] [Q&A] [answered]
I want to make my site multilingual. I have looked into old issues but couldnt figure how. Any help appreciated.

**Comment by mmistakes** [comment, marked as answer]
Jekyll doesn't support this natively, thus neither does this theme. You'd need to hack the Liquid some to do it. There are a few blog posts if you search around describing how to do this with pure Liquid. They all pretty much follow the same strategy… this is one of the better posts on it.

https://www.sylvaindurand.org/making-jekyll-multilingual/ 45

There are also several Jekyll plugins that do similar things and you drop a tag into your layouts:

- http://polyglot.untra.io/
- https://github.com/liamzebedee/jekyll-i18n
- https://github.com/Anthony-Gaudino/jekyll-multiple-languages-plugin
- https://github.com/octopress/multilingual
